src/components/FileRepoSidebar.js

```diff
@@ -178,27 +178,6 @@ export default ({ projectId, style, sqon, ...props }) => (
     </div>
     <Divider />
     <Heading>Data Analysis</Heading>
-    <button
-      css={`
-        border-radius: 19px;
-        background-color: #ffffff;
-        border: solid 1px #cacbcf;
-        font-size: 11px;
-        letter-spacing: 0.2px;
-        color: #008199;
-        padding: 5px 18px 5px 5px;
-        display: flex;
-        align-items: center;
-      `}
-    >
-      <img
-        alt=""
-        src={cavaticaLogo}
-        css={`
-          width: 28px;
-          margin-right: 7px;
-        `}
-      />Export files to Cavatica
-    </button>
+    <CavaticaSidebar {...props} />
```

Comment:
Could you rename this to something like `CavaticaExportButton`. the external link could be moved to be a sibling here.

Reply:
Its not just the button though, this wraps the project list and project adder... its the whole sidebar widget removed from the FileRepoSidebar component, just included inside.

Reply:
`CavaticaExportWidget` maybe? 
sidebar is confusing, cause it's not a sidebar :D

Reply:
Yes. Will rename as Widget. I like Widget.

Reply:
Renamed as suggested.